src/Djot/AST.hs

```diff
@@ -89,6 +90,12 @@ import Data.Data (Data, Typeable)
 import qualified Data.ByteString.Char8 as B8
 import GHC.Generics (Generic)
 import Language.Haskell.TH.Syntax (Lift (..))
+#if __GLASGOW_HASKELL__ <= 902 || ! MIN_VERSION_containers(0,6,6)
```

Comment:
I think it's sufficient just to check containers version, because 0.6.6+ won't build with the old ghc base versions.